broker/lib/controllers/ServiceBrokerApiController.js

```diff
@@ -16,6 +17,8 @@ const ServiceBindingAlreadyExists = errors.ServiceBindingAlreadyExists;
 const ServiceBindingNotFound = errors.ServiceBindingNotFound;
 const ContinueWithNext = errors.ContinueWithNext;
 const UnprocessableEntity = errors.UnprocessableEntity;
+const ETCDLockError = errors.ETCDLockError;
```

Comment:
Can we make it camel case ETCDLockError ==> EtcdLockError

Reply:
Done!